Patch created by brosx "diffPackage" tool

usage:
    diffPackage <tarName> <pkgName> <outName> <patchName> <files>

actual:
    diffPackage "attr-2.4.47.src.tar.gz" "attr" "host-attr" "care-for-gettext-autoconf-param" "\
        po/Makefile \
    ";


===============================================================================
--- a/po/Makefile	2013-05-19 06:53:54.000000000 +0200
+++ b/po/Makefile	2018-01-03 17:55:18.000000000 +0100
@@ -19,6 +19,8 @@
 TOPDIR = ..
 include $(TOPDIR)/include/builddefs
 
+ifeq ($(ENABLE_GETTEXT), yes)
+
 POTHEAD = $(PKG_NAME).pot
 LINGUAS ?= cs de es fr gl nl pl sv
 LSRCFILES = $(LINGUAS:%=%.po) $(POTHEAD)
@@ -36,5 +38,8 @@
 
 install: default
 	$(INSTALL_LINGUAS)
+else
+install:
+endif
 
 install-dev install-lib:
